ports/soh/Ship of Harkinian.sh

```diff
@@ -20,14 +20,14 @@ source $controlfolder/device_info.txt
 [ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 
 # Set variables
-GAMEDIR="$directory/ports/soh"
+GAMEDIR="/$directory/ports/soh"
 > "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
 
 cd $GAMEDIR
 
 # Exports
 export LD_LIBRARY_PATH="$GAMEDIR/libs:/usr/lib":$LD_LIBRARY_PATH
-export SDL_GAMECONTROLLERCONFIG=$sdl_controllerconfig
+[-z "$sdl_controllerconfig" ] && export SDL_GAMECONTROLLERCONFIG=$sdl_controllerconfig
```

Comment:
Is this a necessary change for knulli? We follow some standard launchscripts formats, so if this is necessary here then it will be necessary elsewhere too. Now if this is something that will be resolved etween PortMaster GUI and Knulli, then it does not belong in the launch script.

Reply:
It is a necessary change for RG40XX H on Knulli right now. According to @kloptops it could be resolved with the next PortMaster release because it will add a controller config for RG40XX H.
So maybe wait for that new release instead of fixing it in the launch script.

Reply:
I think it is better to wait and do that properly. This change might break other firmwares for all I know. Can you remove that and do it locally for now, or hide it behind an if clause for cfw_name?